docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1636,8 +1634,8 @@ If a custom realm file has to be imported into Keycloak before running the tests
 quarkus.keycloak.devservices.realm-path=quarkus-realm.json
 ----
 
-Finally, write a test code the same way as it is described in the <<integration-testing-wiremock,Wiremock>> section.
-The only difference is that `@QuarkusTestResource` is no longer needed:
+Finally, write a test code the same way as it is described in the <<code-flow-integration-testing-wiremock,Wiremock>> section.
+The only difference is that `@WithTestResource` is no longer needed:
```

Comment:
What is this about? I don't see this class in the 3.8 https://github.com/quarkusio/quarkus/blob/3.8/test-framework/common/src/main/java/io/quarkus/test/common/WithTestResource.java

Reply:
My bad. I should not have allowed that to slip through. Fixing now.

Reply:
np